src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,172 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    private static class Node<E> {
```

Comment:
make it non static and move to the and of class
Also better to use T in this case (its up to you change it or not)
![image](https://github.com/mate-academy/jv-linked-list/assets/48316721/045d1739-18af-4e8f-a7e3-bee430c3ffd3)